packages/terra-search-field/src/SearchField.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from 'terra-button';
+import IconSearch from 'terra-icon/lib/icon/IconSearch';
+import classNames from 'classnames';
+import './SearchField.scss';
+
+const propTypes = {
+  /**
+   * Placeholder text to show while the search field is empty.
+   */
+  placeholder: PropTypes.string,
+
+  /**
+   * The minimum number of characters to perform a search.
+   */
+  minimumSearchTextLength: PropTypes.number,
+
+  /**
+   * How long the component should wait after input before performing an automatic search.
+   */
+  searchDelay: PropTypes.number,
+
+  /**
+   * A callback to perform search. Sends parameter {String} searchText.
+   */
+  onSearch: PropTypes.func,
+};
```

Comment:
This might be a case where we only add it in the future if it becomes needed.

Unless anyone knows of an immediate need for either one I would just move forward without it. We can passively add it as an additional prop in the future if it ever becomes necessary.